lib/SPIRV/libSPIRV/SPIRVModule.h

```diff
@@ -43,6 +43,8 @@
 #include "LLVMSPIRVOpts.h"
 #include "SPIRVEntry.h"
 
+#include "llvm/ADT/APInt.h"
```

Comment:
In spite of #755, can we just forward-declare `APInt` in this header and put actual include into corresponding `.cpp` file?